raiden/messages.py

```diff
@@ -1848,11 +1849,12 @@ def from_dict(
             nonce=data['nonce'],
             transferred_amount=data['transferred_amount'],
             locked_amount=data['locked_amount'],
```

Comment:
according to RFC 7159, the range for numbers in JSON (which you're ultimately serializing to) is  `[-(2**53)+1, (2**53)-1]`. So for token values, you might want to serialize as `str` and deserialize an `int` as we do elsewhere, e.g. https://github.com/raiden-network/raiden/blob/master/raiden/messages.py#L1583

Quote from the RFC:
> Note that when such software is used, numbers that are integers and
   are in the range [-(2**53)+1, (2**53)-1] are interoperable in the
   sense that implementations will agree exactly on their numeric
   values.